pkg/sql/codegen/pai/tensorflow.go

```diff
@@ -112,6 +112,39 @@ func TFLoadAndExplain(ir *ir.ExplainStmt, session *pb.Session, modelPath string,
 	return code.String(), nil
 }
 
+// TFLoadAndEvaluate generates PAI-TF evaluate program.
+func TFLoadAndEvaluate(ir *ir.EvaluateStmt, session *pb.Session, modelPath string, expn *ExplainRender) (string, error) {
+	var tpl = template.Must(template.New("Evaluate").Parse(tfEvaluateTmplText))
+	ossModelDir := OSSModelURL(modelPath)
+	paiExplainTable := ""
+	if tensorflow.IsPAI() && ir.TmpEvaluateTable != "" {
+		paiExplainTable = ir.TmpEvaluateTable
+	}
+
+	validationMetrics := ""
+	for k, v := range ir.Attributes {
```

Comment:
Can we  use an `if` statement here instead? Like
```go
if v, ok := ir.Attributes["validationMetrics"]; ok && len(ir.Attributes) == 1 {
    validationMetrics = v.(string)
} else if len(ir.Attributes) != 0 {
    // Output diagnostics messages
}
```

Reply:
Done.